Allow transformed URL to sign (#181)
uploadcare/uploadcare-php: README.md

$secureUrlFromApi = $api->file()->generateSecureUrl($file);
```

### Secure delivery for transformed images

> You can set the custom ACL to Secure URL for images which were transformed with [Image Transformations](https://uploadcare.com/docs/transformations/image/). Set Image UUID and transformation part to `generateSecureUrl` method. Like this:

Reviewer: ```suggestion
You can set a custom ACL on a Secure URL with [Image Transformations](https://uploadcare.com/docs/transformations/image/)
applied. Just add an Image UUID with transformations to the `generateSecureUrl` method:
```
